repos/system_upgrade/common/actors/preparepythonworkround/libraries/workaround.py

```diff
@@ -18,6 +19,8 @@ def _get_orig_leapp_path():
 
 def apply_python3_workaround():
     py3_leapp = os.path.join(LEAPP_HOME, 'leapp3')
+    if os.path.exists(LEAPP_HOME):
+        shutil.rmtree(LEAPP_HOME, ignore_errors=True)
```

Comment:
hmm...it's a nitpick. but...I know we do not expect errors during the removal. I would add there a note or something, that in case we ever see an issue happening here, we should create a separate actor to handle it before the rpm transaction is performed. and possibly remove the "ignore_errors" in such a case. I mean, in case this will crash, most likely we will see crash when we try to run leapp after the rpm transaction...

Reply:
added a logger